EDI/Operation/05_ManageDataSources.md

```diff
@@ -0,0 +1,156 @@
+[!!Exports](../UserInterface/01_Exports.md)
+
+# Manage data sources
+Specify the data sources from which you want to retrieve the string attribute data that you have defined both in the header file settings and the line items file settings.   
+For example, you have defined a string attribute to determine the invoice document number. By specifying a data source here, you determine the data source from which the invoice document number is to be retrieved.    
+If required, you can specify here multiple data sources from which you can retrieve data from different sources, such as invoice data, customer data, and product data. In the next step, you can map a data source field with the related data (source attribute) with the EDIFACT field (destination attribute). The EDIFACT fields are automatically provided after you have saved a string attribute allocation in the file settings.   
```

Comment:
Vorschlag: If required, you can specify here multiple data sources to retrieve data from different sources, such as... / You can specify here multiple data sources if you want/need to retrieve data from different sources, such as...

Reply:
done